06_deploy_bootstrap_vm.sh

```diff
@@ -6,63 +6,25 @@ source ocp_install_env.sh
 source common.sh
 source utils.sh
 
-# FIXME this is configuring for the libvirt backend which is dev-only ref
-# https://github.com/openshift/installer/blob/master/docs/dev/libvirt-howto.md
-# We may need some additional steps from that doc in 02* and also to make the
-# qemu endpoint configurable?
 if [ ! -d ocp ]; then
     mkdir -p ocp
-    cat > ocp/install-config.yaml << EOF
+    cat > ocp/baremtal-install-config.yaml << EOF
 apiVersion: v1beta3
 baseDomain: ${BASE_DOMAIN}
 metadata:
   name: ${CLUSTER_NAME}
 platform:
-  libvirt: {}
+  baremetal: {}
```

Comment:
I know we discussed this a bit last week, but I wonder if we ran Ironic on the bootstrap VM if we could just enter the node details here, e.g a list with the IPMI creds etc.  That doesn't satisfy the discovery workflow, but it could provide a simple way to inject the baremetal inventory via an existing openshift-install interface (vs the new create nodes thing I mentioned).

Reply:
Yes, absolutely

Reply:
I had another thought on this, if we wanted to avoid running Ironic in two places, we could optionally pass in an existing endpoint (e.g a URL pointing to a bridge the bootstrap VM can access), this can be the already-running Ironic used for discovery, and if none is provided we go ahead and launch that same container on the bootstrap node?

Reply:
There's already minimal support for `platform: none: {}` in upstream installer btw

Reply:
The point about external Ironic is captured here - https://github.com/metalkube/kni-installer/blob/master/docs/dev/baremetal.md

The point about passing node details via the platform configuration is definitely the plan